samples/apkman/README.md

```diff
@@ -0,0 +1,305 @@
+# oeapkman Sample
+
+This sample demonstrates installing and using [sqlite](https://sqlite.org/index.html)
+database static library via `oeapkman`. `sqlite` is used to create, populate
+and query an in-memory database within an enclave.
+
+# oeapkman Overview
+
+`oeapkman` is a tool for installing and using [Alpine Linux](https://www.alpinelinux.org/)
+static libraries in enclaves. Currently, `oeapkman` is Linux only.
+
+## Why Alpine Linux?
+
+OE SDK uses [MUSL](https://musl.libc.org/) as its enclave C library.
+Most Linux distributions use GLIBC as the native C library.
+MUSL is currently not 100% binarycompatible with GLIBC as explained here:
+- https://www.musl-libc.org/faq.html
+- https://wiki.musl-libc.org/functional-differences-from-glibc.html
+
+Libraries that successfully compile with GLIBC may not always
```

Comment:
```suggestion
Libraries that are successfully compiled with GLIBC may not always be
```

Reply:
Retaining the original tense; seems cleaner.